app/views/devise/registrations/new.html.erb

```diff
@@ -11,8 +11,9 @@
   <div class="field">
     <%= f.label :password %>
     <% if @minimum_password_length %>
-    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <em>(<%= @minimum_password_length %> to 10 characters)</em>
```

Comment:
should the logic to set minimum be in the view? Could it be moved to the controller, which is responsible for telling the view what to do?

Reply:
As discussed in person: this is a the part on the Devise set-up. I would prefer not to dive in to modifications since we have already tried to go this path with Danny and it proved quite difficult to modify in any way - at this level of understanding.